mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/MongoDBQueryDescriptorBuilder.java

```diff
@@ -157,6 +158,11 @@ public boolean setOrderBy(String orderBy) {
 		return true;
 	}
 
+	public boolean setCount(boolean count) {
+		this.count = count;
+		return true;
+	}
+
```

Comment:
@Krerg One quick question: Is this field still necessary?
(Non need to update the PR, I will do it)